lib/ProductOpener/Images.pm

```diff
@@ -961,6 +962,28 @@ sub process_image_crop($$$$$$$$$$) {
 	return $product_ref;
 }
 
+sub process_image_unselect($$) {
+
+	my $code = shift;
+	my $id = shift;
+	
+	my $path = product_path($code);
+		
+	print STDERR "Images.pm - process_image_unselect - id: $id\n";
+			
+	# Update the product image data
+	my $product_ref = retrieve_product($code);
+	defined $product_ref->{images} or $product_ref->{images} = {};
+	if (defined $product_ref->{images}{$id}) {
+		delete $product_ref->{images}{$id};
+	}
+
+	store_product($product_ref, "unselected image $id");
```

Comment:
We could, but then we need to do something else than the $comment = lang($comment) if $comment; as there's the id of the image in the comment. Not sure if it's worth it.